modules/core/src/shaderlib/project/project.glsl.js

```diff
@@ -89,8 +91,13 @@ vec4 project_offset_(vec4 offset) {
 // Projecting positions - non-linear projection: lnglats => unit tile [0-1, 0-1]
 //
 vec2 project_mercator_(vec2 lnglat) {
+  float x = lnglat.x;
+  if (project_uWrapCoordinates) {
```

Comment:
This PR is not about handling out-of-range coordinates in the *source data*, but an inherited problem in web mercator projection. See my comment below.

Reply:
> This PR is not about handling out-of-range coordinates in the source data, but an inherited problem in web mercator projection. See my comment below.

Yes that is why I suggested not using modular arithmetic, since that changes the behavior of out of range coordinates.